armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -330,6 +486,23 @@ def _conserveComponentDensity(self, b, oldHeight, oldVolume):
                     c.setNumberDensity(key, c.getNumberDensity(key) / growth)
 
 
+def _getComponent(b, componentFlag):
```

Comment:
Aside from the error messages being different, is this functionally different from `composites.getComponent()`?

Reply:
Nope! Great catch.
https://github.com/terrapower/armi/pull/931/commits/eec4718236c18f7f0ff8f2d22411e8c2c34dd019